src/config.py

```diff
@@ -89,6 +89,7 @@ class Secrets:
         SECRET_INTERNAL_LABEL = "internal-secret"
         SECRET_DELETED_LABEL = "None"
         SECRET_KEYFILE_NAME = "keyfile"
+        MAX_PASSWORD_LENGTH = 4096
```

Comment:
noob question, how did you get this number

Reply:
from a discussion with @delgod a while ago when we had similar issue on VM charm